index.js

```diff
@@ -38,8 +38,51 @@ alfy.alfred = {
 
 alfy.input = process.argv[2];
 
-alfy.output = arr => {
-	console.log(JSON.stringify({items: arr}, null, '\t'));
+const isDefined = x => x !== null && x !== undefined;
+
+const wrapArg = item => {
+	const alfredworkflow = {arg: item.arg, variables: item.variables};
+	const arg = JSON.stringify({alfredworkflow});
+	const newItem = Object.assign({}, item, {arg});
+	delete newItem.variables;
+	return newItem;
+};
+
+const formatMods = item => {
+	const copy = Object.assign({}, item);
+
+	Object.keys(item.mods).forEach(modKey => {
+		const modItem = item.mods[modKey];
+		copy.mods[modKey] = wrapArg(modItem);
+	});
+
+	return copy;
+};
+
+// see https://www.alfredforum.com/topic/9070-how-to-workflowenvironment-variables/
+// for documentation on setting environment variables from Alfred workflows
+const format = item => {
+	if (!isDefined(item)) {
+		return item;
+	}
+
+	if (isDefined(item.variables)) {
+		item = wrapArg(item);
+	}
+
+	if (isDefined(item.mods)) {
+		item = formatMods(item);
+	}
+
+	return item;
+};
+
+alfy.output = items => {
+	if (Array.isArray(items)) {
+		items = items.map(format);
```

Comment:
It's better to use `items.map(item => format(item))`. The reason for this is that the `map` callback returns multiple parameters. This results in sending all those parameters to `format` which may result in bugs in the future if for instance format starts excepting additional input arguments.